### PR TITLE
Add an OEP-2 compliant openedx.yaml file

### PR DESCRIPTION
**Pull request by cpennington**
This adds an `openedx.yaml` file, as described by OEP-2:
http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html

The data in this file was transformed from the contents of
edx/repo-tools-data:repos.yaml

### CODE REVIEW

**Comment by cpennington** [comment]
@edx/teaching-and-learning: I've tagged your team as the owner of this repo. Ok to merge?

**Comment by cahrens** [comment]
Can we just close out this repo instead?

**Comment by andy-armstrong** [comment]
@cpennington @cahrens maybe we need a deprecated status in the yaml file. Or is it better to just delete the yaml file once a repo is no longer active.

**Comment by cpennington** [comment]
I've marked the repo as `obsolete`.

**Comment by andy-armstrong** [comment]
Nice (I guess I should have read OEP-2 in full!). 👍

**Comment by cahrens** [comment]
But is there a way for us to ultimately delete these repos?

**Comment by andy-armstrong** [comment]
We could delete repos, but it seems like a bad idea to me. Marking them as obsolete seems good enough.

https://help.github.com/articles/deleting-a-repository/

**Comment by cpennington** [comment]
Yeah, we shouldn't delete repos.

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+oeps: {}
+owner: edx/teaching-and-learning
+obsolete: True
```
Comment (robrap) [inline openedx.yaml]: @e0d: I think Gemnasium reports on this repo.

1. Do you want a ticket to respect this "obsolete" status in Gemnasium, either manually or automatically?
2. Should we require that the README also gets updated with a warning for obsolete repos that their security, or lack there of, is not being tracked and it is likely unsafe to use?  Or, do we just want to clearly document this along with our definition of what it means to be an "obsolete" repo?

FYI: @cpennington

**Comment by andy-armstrong** [comment]
@cahrens @nasthagiri and I were just discussing this. We could create a new org for obsolete repos (edx-legacy?) and move repos like this one there. This would allow us to unclutter our current org. I guess this would be a question for @jbarciauskas.

Here's how to transfer a repo from one org to another:

https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/

**Comment by andy-armstrong** [comment]
👍  to merge despite the CI failures, since this repo is obsolete.

**Comment by pmitros** [comment]
@cahrens @cpennington @andy-armstrong If this repo is, indeed, completely obsolete or deprecated, do feel free to transfer it to github.com/pmitros. That would decouple it from edX, but github would provide a forwarding link.

**Pull request merged**